script/core/monitors/MarketHealth.s.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier:	AGPL-3.0
```

Comment:
Maybe `script/periphery`? After the repo split they will probably not be in mangrove-core

Reply:
But the script is about testing the state of mangrove-core. So not sure about this point